README.md

```diff
@@ -66,10 +66,12 @@ Configuration options can be specified on the command line such as
 `-DdependencyExcludes="io.github.mfoo:*"` or as part of the plugin
 configuration in `pom.xml`.
 
-| Option               | Description                                                 | Format                          |
-|----------------------|-------------------------------------------------------------|---------------------------------|
-| `dependencyExcludes` | Ignore certain dependencies                                 | `io.github.mfoo:*,org.apache:*` |
-| `maxLibYears`        | Cause the build to fail if dependencies are older than this | `4`                             |
+| Option                 | Description                                                 | Format                          |
+|------------------------|-------------------------------------------------------------|---------------------------------|
+| `dependencyExcludes`   | Ignore certain dependencies                                 | `io.github.mfoo:*,org.apache:*` |
+| `maxLibYears`          | Cause the build to fail if dependencies are older than this | `4`                             |
+| `reportFile`           | The path to report file                                     | `target/libyear-report.csv`     |
```

Comment:
This param is now the name of the file inside the target directory, so we can remove `target/` from the example